packages/components/src/Slider/Slider.component.js

```diff
@@ -33,6 +34,31 @@ export function getSelectedIconPosition(icons, value, min, max) {
 	return Math.round(value / interval);
 }
 
+/**
+ * This function allow to get the actions components
+ * @param {array} actions
+ * @param {number} value
+ * @param {number} min
+ * @param {number} max
+ */
+export function getActions(actions, value, min, max) {
```

Comment:
The component has already a event onChange. We have here an implementation very coupled with CMF. 

When we click on an icon, he should just trigger an event onChange with the value.

Reply:
Erf didn't know the icon can render a button... 
Love the if that changes the rendering of a component... is so handy.
So yeah you're right for the implem, back to work

Reply:
Hmmm another problem is that we have less posibility with the icon than the action. No tooltip for example.